config/initializers/rack_attack.rb

```diff
@@ -0,0 +1,23 @@
+Rack::Attack.cache.store = ActiveSupport::Cache::MemoryStore.new
+Rack::Attack.throttled_response = ->(env) { [429, {}, [ActionView::Base.new.render(file: 'public/429.html')]] }
+
+ActiveSupport::Notifications.subscribe('rack.attack') do |name, start, finish, request_id, req|
+  Airbrake.notify Exception.new(message: "#{req.ip} has been rate limited for url #{req.url}", data: req.env['rack.attack.throttle_data'])
```

Comment:
I'll readily admit this might not be the optimal info to shoot through to airbrake, but once we start getting some of these come through in prod I'll have a much better idea what's useful to put here.